src/providers/hollisterco/scraper.ts

```diff
@@ -0,0 +1,174 @@
+import Product from '../../entities/product'
+import Scraper from '../../interfaces/scraper'
+import screenPage from '../../utils/capture'
+
+const scraper: Scraper = async (request, page) => {
+  page.setUserAgent(
+    'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.75 Safari/537.36',
+  )
```

Comment:
por que hace falta este UA? no anda con el default?

Reply:
El UA default tiene la string HeadlessChrome, si lo corremos headless es necesario cambiar el Header. Más info: https://jsoverson.medium.com/how-to-bypass-access-denied-pages-with-headless-chrome-87ddd5f3413c